src/pages/dataTableUtilities/reducer/tableReducers.js

```diff
@@ -144,9 +159,14 @@ export const toggleShowFinalised = state => {
   return { ...state, data: sortedData, showFinalised: newShowFinalisedState, searchTerm: '' };
 };
 
-export const showIndicators = state => ({ ...state, showIndicators: true });
-
-export const hideIndicators = state => ({ ...state, showIndicators: false });
+export const toggleIndicators = state => {
+  const { showIndicators } = state;
+  return {
+    ...state,
+    showIndicators: !showIndicators,
+    searchTerm: '',
```

Comment:
cant test this at the moment could be wrong:

If you: 1) filter with the search bar while toggled on items 2) toggle to indicators 3) toggle back to items - are all the items still filtered, with nothing in the search bar?

Similar with sorting, but that might not matter to much.

Reply:
Tested and works. I believe changing the `searchTerm` prop programmatically triggers the `SearchBar` `onChangeText` callback, so the behaviour you're describing doesn't occur.